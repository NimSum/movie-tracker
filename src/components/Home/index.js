import React from 'react';
import Movies from '../../containers/Movies';
import Banner from '../Banner';

const Home = () => (
  <section className="home-section">
    < Banner />
    < Movies />
  </section>
)

export default Home;