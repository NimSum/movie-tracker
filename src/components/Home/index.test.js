import React from 'react';
import { shallow } from 'enzyme';
import Home from './index'

describe('Home', () => {
  let wrapper = shallow(< Home />)
  it('should match component snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  })
})