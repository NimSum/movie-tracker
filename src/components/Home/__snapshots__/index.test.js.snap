// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should match component snapshot 1`] = `ShallowWrapper {}`;
